updater/spec/fixtures/bundler_grouped/original/Gemfile.lock

```diff
@@ -15,4 +15,4 @@ DEPENDENCIES
   ungrouped-dummy-pkg-c (~> 1.0.0)
 
 BUNDLED WITH
-   1.14.6
```

Comment:
🤔 I'm kinda surprised we had this using `1.14.6` and never updated it previously... was it checking some error condition such as an older outdated version?

Also, I'm not a bundler expert, but I'm surprised that nothing else changed in the lockfile... did you actually try regenerating this file with bundler 2.4.13, or did you just manually bump this text string?

Reply:
Yes, actually I manually did the change. I can try to update it by using bundler but since we are using dummy pkg not sure if it is going to work.

Reply:
I think that's was just checking grouping and I didn't think much regarding having difference between bundler v1, and v2.